devcareer/code-jammers-backend
# #174566481 Setup tdd environment with mocha/chai

## What does this PR do
   This PR sets up a test driven development (tdd) environment with Mocha/chai for Know Africa.

## Description of tasks to be completed?
 - Set up tdd environment with Mocha/chai to test cod
  - Install Mocha, chai, chai-http from npm
  - Configure package.json with a test script

## How should this be manually tested/checked?
- Clone the PR
- Cd into the branch `ch-set-up-tdd-environment-174566481`
- Install the dependencies using `npm install`
- Run the test on the terminal using `npm run test`
It should show 1 passing

## What are the relevant pivotal tracker stories?
[#174566481](https://www.pivotaltracker.com/story/show/174566481)
